Intersect (Core)/Network/Packets/Server/FadePacket.cs

```diff
@@ -0,0 +1,19 @@
+using MessagePack;
+
+namespace Intersect.Network.Packets.Server;
+
+[MessagePackObject]
+public class FadePacket : IntersectPacket
+{
+    public FadePacket(bool fadeOut)
+    {
+        FadeOut = fadeOut;
+    }
+
+    public FadePacket()
+    {
+    }
+
+    [Key(0)]
+    public bool FadeOut { get; set; }
```

Comment:
With my enum suggestion this would become an enum

Reply:
Done